SAR_retrieval_funcs.py

```diff
@@ -393,8 +393,8 @@ def invert_field(svv, svh, theta, prior_mean, prior_sd, gamma, s2_lai):
 
 
 
-    
-    
+
+
```

Comment:
Out of curiosity... Did you run the tests? I think that they may need changing to account for the new angle contribution.

Reply:
no I haven't checked that

Reply:
Done